index.html

```diff
@@ -24,14 +24,167 @@ <h2 class="intro-title">Hey there. I'm Juan Carlos</h2>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+                <div class="card-container">
+                    <div class="card-info">
+                        <p class="card-title">Multi-Post Stories Gain+Glory</p>
+                        <ul class="skills-container">
+                            <li class="work-skill">Ruby on rails</li>
+                            <li class="work-skill">css</li>
+                            <li class="work-skill">JavaScript</li>
+                            <li class="work-skill">html</li>
+                        </ul>
+                        <button type="button" class="default-button">See Project</button>
+                    </div>
+                </div>
+
+            </div>
+        </section>
+
+        <section class="about-container">
+            <h2 class="about-title">About me</h2>
+            <p class="about-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <button type="button" class="default-button">Get My Resume</button>
```

Comment:
[ Required ] Please note that the goal of the exercise is to create the project as illustrated in the Figma template, the `About section` on the Figma template has a background image on the `bottom-left` corner of the design please see the screenshots below and rectify this.

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/198440882-2043439c-33b9-4183-8330-39046b21aeed.png" width="300px" />


## Current implementation
<img src="https://user-images.githubusercontent.com/16392046/198439529-fbd877c6-e8af-4f65-a87b-eb056db95e70.png" width="300px" />